bindings/c/local.mk

```diff
@@ -35,6 +35,13 @@ ifeq ($(PLATFORM),linux)
   fdb_c_tests_LIBS += -lpthread
 endif
 
+ifeq ($(PLATFORM),freebsd)
+  fdb_c_CFLAGS += -fPIC
+  fdb_c_LIBS += -lm -lpthread -lrt -lexecinfo -ldevstat -lkvm
```

Comment:
It seems like a lot of these libraries are repeated throughout the various local.mk files.  Is there any amount of this that you could just move to the main Makefile as a default set of LIBS to use on FreeBSD?

Reply:
Yes, that was the last minute workaround and was going to ask if LIBS could/should be put in the Makefile- this answers that question.